pallets/moonbeam-lazy-migrations/src/lib.rs

```diff
@@ -48,7 +53,7 @@ pub mod pallet {
 
 	/// Configuration trait of this pallet.
 	#[pallet::config]
-	pub trait Config: frame_system::Config {}
+	pub trait Config: frame_system::Config + pallet_democracy::Config {}
```

Comment:
We can't depend on the pallet democracy because this pallet will be removed before the lazy migration